index.rst

```diff
@@ -318,56 +318,101 @@ being performed to clarify the classifications of data and services together.
++------------------------+------------------------+------------+-------------------------+
+| Image Access           | read:image/md          | Yes        | img_md                  |
+| (Metadata)             |                        |            |                         |
++------------------------+------------------------+------------+-------------------------+
+| Table Access (DR,      | read:tap               | Yes        | tap                     |
+| Alerts)                |                        |            |                         |
++------------------------+------------------------+------------+-------------------------+
+| Table Access           | read:tap/efd           | Yes        | tap_efd                 |
+| (Transformed EFD)      |                        |            |                         |
++------------------------+------------------------+------------+-------------------------+
+| Table Access (User and | read:tap/user,         | Yes        | tap_usr                 |
+| Shared)                | write:tap/user         |            |                         |
++------------------------+------------------------+------------+-------------------------+
+| User Query History     | read:tap/history       | Yes        | tap_hist                |
++------------------------+------------------------+------------+-------------------------+
+| File/Workspace Access  | read:workspace         | Yes        | ws                      |
++------------------------+------------------------+------------+-------------------------+
+| File/Workspace Access  | read:workspace/user,   | Yes        | ws_usr                  |
+| (User/Shared)          | write:workspace/user   |            |                         |
++------------------------+------------------------+------------+-------------------------+
+| Portal                 | exec:portal            | No         | portal                  |
++------------------------+------------------------+------------+-------------------------+
+| Notebook               | exec:notebook          | No         | nb                      |
++------------------------+------------------------+------------+-------------------------+
+
 Tokens
 ======
 
 Broadly speaking, there are two main types of tokens in the LSST DM system. Tokens whose primary use
-are for identity, which are issued from CILogon, and tokens whose primary use are for checking
-capabilities. Identity tokens are roughly equivalent to X.509 certificates; they include information
-about the user identity, including the username for the LSST account and/or the UNIX UID, and group
-memberships, in addition to a cryptographic signature for verifying the token integrity using public
-key encryption.
+are for identity, which are similar to those issued from CILogon, and tokens whose primary use are
+for checking capabilities. Identity tokens are roughly equivalent to X.509 certificates; they
```

Comment:
You must distinguish between tokens and X.509 certificates here.  I think adding "but can have a limited lifetime and do not require complex certificate authority handling" or something like that would help.

Reply:
Some additional differences were covered in another section below. I augmented both sections to reinforce the differences.